invenio_rdm_records/records/mappings/v6/rdmrecords/drafts/draft-v4.0.0.json

```diff
@@ -163,7 +163,18 @@
                   }
                 },
                 "role" : {
-                  "type" : "keyword"
+                  "type": "object",
+                  "properties" : {
+                    "@v" : {
+                      "type" : "keyword"
+                    },
+                    "id" : {
+                      "type" : "keyword"
+                    },
+                    "title" : {
+                      "type" : "object"
```

Comment:
Do we need `dynamic: true` since the schema is strict on the top? see languages

Reply:
Hmm it makes sense in case we want to add fields dinamically to the object, will change it for the vocabs :+1:  Thanks!